src/sass/_shame.scss

```diff
@@ -292,3 +292,7 @@ form {
 select {
   background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAcCAYAAABh2p9gAAAAAXNSR0IArs4c6QAAAPFJREFUSA3tlr0OAUEUhZdCRSQSBVFsQuNtPCS9t7CFRqNAFAoPoeE7m72FiTHWXp2THPN3z7eJ3Z3ZLEtrTsmxsvqNNCN9xffK6mvuK+WkLthg1mpOa7U0ofqMDRK2WlPNRxpRdcAhJByrRrVvNWR1j8NwbKxaZV5qwOwOx8KxeWWUfVKf0RbHQql5ZcUo1eN3g1Oh1LoYYmUrB5hdbNkGNhXVSbMWoDFe4E5D6I38uiHjH+cf+MlNKQDbc9S0LfQcnhzvVcnqAvR69cQq5bo5GNR1+zKo6wZrUNcjwKCuh5RBczpux6hBXQ96g9b6FHkA0+zeRS2oYfAAAAAASUVORK5CYII=);
 }
+
+.login-container {
+  display: inline-block;
```

Comment:
What problem did this solve?

Reply:
The issue is more obvious on the disability increase wizard: 

Corrected (Left) vs. Uncorrected (Right)
<img width="1437" alt="screen shot 2018-04-04 at 7 30 50 pm" src="https://user-images.githubusercontent.com/16051172/38344523-e1453c46-383e-11e8-8813-835a7cd78c64.png">

but there is a slight excess of whitespace corrected here by including this:

Uncorrected (Left) vs. Corrected (Right)
<img width="1439" alt="screen shot 2018-04-04 at 7 27 54 pm" src="https://user-images.githubusercontent.com/16051172/38344510-cda256e2-383e-11e8-8ef2-738b1c07939d.png">